src/features/basic/slider-improvements/slider-improvements.ts

```diff
@@ -0,0 +1,130 @@
+import $style from './slider-improvements.module.css';
+import SliderButton from '@src/features/basic/slider-improvements/SliderButton.vue';
+import { clickElement } from '@src/util';
+
+function onTileReady(tile: PrunTile) {
+  sliderMinMaxButtons(tile);
+  sliderTableBulkMinMax(tile);
+}
+
+function sliderTableBulkMinMax(tile: PrunTile) {
+  subscribe($$(tile.anchor, 'table'), table => {
+    const headerRow = _$(table, 'thead')?.firstElementChild;
+    if (!headerRow) return;
+    const sliders = _$$(table, 'rc-slider');
+    if (sliders.length > 0) {
+      const sliderColumns = new Map<number, HTMLElement[]>();
+      sliders.forEach(slider => {
```

Comment:
Code style for this project: never use `.forEach`
```suggestion
      for (const slider of sliders) {
```